### DISCUSSION TITLE
When v1.20.2 will be released?

### DISCUSSION THREAD

**Discussion by jovezhong** [testcontainers/testcontainers-java#9217] [General]
1.20.1 was release July 31. 

Not sure when the next release will be available. Is there any project/page to check the rough release schedule? Thanks.

**Comment by jovezhong** [comment]
If the 1.20.2 release is not coming soon, how to get an nightly/weekly build(via maven) with the latest PR merges? I'd like to use the latest/unreleased testcontainer in my pom.xml for CI automation.

**Reply by eddumelendez** [reply]
Hi, I'm planning to release a new version in the next days. Regarding to unreleased version, see https://java.testcontainers.org/jitpack_dependencies/
